mentors/clojure/h.json

```diff
@@ -1 +1,8 @@
-[]
+[{
+  "github_username": "holyjak",
+  "name": "Jakub Holy",
+  "link_text": "Holy on Dev",
+  "link_url": https://blog.jakubholy.net/,
+  "avatar_url": null,
+  "bio": " I’m a full stack Clojure(Script) developer coming originally from the JVM/JS land, coding professionally since 2005. I love Clojure for its interactive development, productivity, and simplicity."
```

Comment:
```suggestion
  "bio": "I’m a full stack Clojure(Script) developer coming originally from the JVM/JS land, coding professionally since 2005. I love Clojure for its interactive development, productivity, and simplicity."
```